### PR TITLE
Used cert-manager's certificate

### PR DESCRIPTION
**Pull request by Jaryllan**
- Used cert-manager's certificate for argocd, harbor and elasticsearch
- Fix logging role requires authentication in new elastic stack version

### CODE REVIEW

File: roles/argocd/templates/certificate.j2 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nginx-cert
+  namespace: argocd
+spec:
+  dnsNames:
+  - {{ domain }}
```
Comment (serenagrl) [inline roles/argocd/templates/certificate.j2]: Please standardize the indentation for list items in yaml.

File: roles/argocd/templates/ingress.j2 [hunk 2]
```diff
@@ -19,4 +19,10 @@ spec:
               service:
                 name: argocd-server
                 port:
-                  name: https
\ No newline at end of file
+                  name: https
+{% if cm_status.stdout == "Active" %}
+  tls:
+  - hosts:
```
Comment (serenagrl) [inline roles/argocd/templates/ingress.j2]: Please standardize the indentation for list items in yaml.

File: roles/harbor/templates/certificate.j2 [hunk 3]
```diff
@@ -0,0 +1,13 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nginx-cert
+  namespace: harbor-system
+spec:
+  dnsNames:
+  - core.{{ domain }}
```
Comment (serenagrl) [inline roles/harbor/templates/certificate.j2]: Please standardize the indentation for list items in yaml.

File: roles/harbor/templates/certificate.j2 [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  - notary.{{ domain }}
```
Comment (serenagrl) [inline roles/harbor/templates/certificate.j2]: Please standardize the indentation for list items in yaml. Please change for all other instances.

**Review by serenagrl — COMMENTED** [review]
Please make the necessary changes.

File: roles/argocd/templates/certificate.j2 (hunk 1 above)
Comment (Jaryllan) [reply roles/argocd/templates/certificate.j2]: Fixed

File: roles/argocd/templates/ingress.j2 (hunk 2 above)
Comment (Jaryllan) [reply roles/argocd/templates/ingress.j2]: Fixed

File: roles/harbor/templates/certificate.j2 (hunk 3 above)
Comment (Jaryllan) [reply roles/harbor/templates/certificate.j2]: Fixed

File: roles/harbor/templates/certificate.j2 (hunk 4 above)
Comment (Jaryllan) [reply roles/harbor/templates/certificate.j2]: Fixed

**Pull request merged**